substrate/client/src/client.rs

```diff
@@ -335,6 +335,7 @@ impl<B, E, Block> Client<B, E, Block> where
 
 		let is_new_best = header.number() == &(self.backend.blockchain().info()?.best_number + One::one());
 		trace!("Imported {}, (#{}), best={}, origin={:?}", hash, header.number(), is_new_best, origin);
+		telemetry!("block.import"; "height" => { let n: u64 = header.number().as_(); n }, "best" => ?hash, "is_new_best" => is_new_best, "origin" => ?origin);
```

Comment:
could we please wrap this 158char line ;) ?

Reply:
This should be called after `import_lock` is released in `import_block`.